Xamarin.Android.sln

```diff
@@ -1,6 +1,8 @@
 ﻿
 Microsoft Visual Studio Solution File, Format Version 12.00
-# Visual Studio 2012
+# Visual Studio 15
```

Comment:
As mentioned on the call, I think `Xamarin.Android.sln` can be left unchanged, and this should instead modify `Xamarin.Android-Tests.sln`, as the timing-related functionality is only useful alongside the tests.